apps/frontend/src/app/Components/GeneralAutocomplete.tsx

```diff
@@ -17,22 +30,81 @@ export default function GeneralAutocomplete<T extends string>({ options, valueKe
       return newValue !== null && onChange(newValue.key)
     }}
     isOptionEqualToValue={(option, value) => option.key === value.key}
-    renderInput={props => <SolidColoredTextField

```

Comment:
We lose the colored background that is throughout the site with this change. Did you intend this?

Reply:
yes the colored field background does not look good in autocompletes, I found.

Reply:
might need to look at how autocomplete looks with ArtifactCard